_data/students.yml

```diff
@@ -189,3 +189,8 @@
   image: KBL.jpg
   facebook: SarathNandimandalam
   bio: AI maniac.
+ 
+- name: praful tondomker 
+  github: https://github.com/SSPrafulTondomker
+  image: 
```

Comment:
please provide an image in the images folder, any image would be nice.
provide its name over here